website/docs/r/autoscale_setting.html.markdown

```diff
@@ -10,6 +10,8 @@ description: |-
 
 Manages an AutoScale Setting which can be applied to Virtual Machine Scale Sets, App Services and other scalable resources.
 
+~> **NOTE:** This resource has been deprecated in favour of the `azurerm_monitor_autoscale_setting` resource and will be removed in the next major version of the AzureRM Provider. The new resource shares the same fields as this one, and information on migrating across [can be found in this guide](../guides/migrating-between-renamed-resources.html).
```

Comment:
Should we specify in v2.0 instead of next major version here?

Reply:
this is covered by the upgrade guide, so I'd suggest we don't for the moment